include/rmm/mr/device/cuda_async_memory_resource.hpp

```diff
@@ -44,6 +44,17 @@ namespace rmm::mr {
  */
 class cuda_async_memory_resource final : public device_memory_resource {
  public:
+  /**
+   * Flags for specifying particular handle types
+   */
+  enum class allocation_handle_type {
+    none                  = 0x0,  ///< Does not allow any export mechanism.
+    posix_file_descriptor = 0x1,  ///< Allows a file descriptor to be used for exporting. Permitted
+                                  ///< only on POSIX systems.
+    win32     = 0x2,              ///< Allows a Win32 NT handle to be used for exporting. (HANDLE)
+    win32_kmt = 0x4  ///< Allows a Win32 KMT handle to be used for exporting. (D3DKMT_HANDLE)
```

Comment:
Are we adding Windows support (or should this already work now)? Is this being tested somehow (or is there a plan to do so)? Understand if this might still be coming together. Just was curious as I wasn't aware of Windows support previously

Reply:
No plans. It might work, but this was added just for 1:1 matching of the CUDA API, which does support Windows.